bindings/python/iota_sdk/types/token_scheme.py

```diff
@@ -1,30 +1,25 @@
 # Copyright 2023 IOTA Stiftung
 # SPDX-License-Identifier: Apache-2.0
 
+from dataclasses import dataclass
+from iota_sdk.types.common import HexStr
+
+
+@dataclass
 class TokenScheme():
-    def __init__(self, minted_tokens=None, melted_tokens=None, maximum_supply=None):
-        """Initialize TokenScheme
-        
-        Parameters
-        ----------
-        minted_tokens : int
-        melted_tokens : int
-        maximum_supply : int
-        """
-        self.type = 0
-        
-        self.mintedTokens = minted_tokens
-        self.meltedTokens = melted_tokens
-        self.maximumSupply = maximum_supply
+    mintedTokens: HexStr
+    meltedTokens: HexStr
+    maximumSupply: HexStr
+    type: int = 0
 
     def as_dict(self):
         config = dict(self.__dict__)
         
-        if 'mintedTokens' in config:
+        if isinstance(config['mintedTokens'], int):
```

Comment:
Now I know that Pythoneans are crazy if they like name like: `isinstance`. You must be kidding me!!!